changelog.md

```diff
@@ -1,5 +1,11 @@
 # Changelog
 
+## v1.3.2 | 2023-12-12
```

Comment:
```suggestion
## v1.3.2 | 2023-12-13
```

Reply:
Oh good catch, I think pycharm autofilled that for me. It should actually be `t.b.d.` since we don't know when we'll release this patch yet.